capsules/extra/src/touch.rs

```diff
@@ -177,7 +177,7 @@ impl<'a> Touch<'a> {
                 }
                 ScreenRotation::Rotated270 => {
                     mem::swap(&mut width, &mut height);
-                    (width as u16 - touch_event.y as u16, touch_event.x)
+                    (width as u16 - touch_event.y, touch_event.x)
```

Comment:
This one is a little weirder as I think `touch_event.y` is still being coerced to a `u16` but _only_ becuase the `width` in the LHS has been coerced. This feels fragile. Maybe better to split into separate lines / temporary variables?

Reply:
Seems that `touch_event.y` is actually a `u16`: https://github.com/tock/tock/blob/0a2b74eba355dba42907fb6f5d693166ad4539be/kernel/src/hil/touch.rs#L35

AFAIK, Rust (still) does not coerce types.